script/bootstrap

```diff
@@ -8,17 +8,25 @@
 
 
 cd "$(dirname "${0}")/.."
-source services.ini
+eval "$(cat services.ini  | ./ini_parser.py)"
 cd services
 top=$(pwd)
+
 mkdir -p log
-for service in $SERVICES; do
+
+for repo in ${SERVICES[@]}; do
+    IFS="@"
+    set $repo
+    service=$1
+    version=$2
+
     cd $top
-    echo "Cloning $service"
-    git clone -q -o github github.com:dod-ccpo/$service
+    echo "Cloning $service at $version"
+    curl -sSL https://github.com/dod-ccpo/$service/archive/$version.tar.gz | tar xvz
+    mv $service-$version $service
     # "script/setup is typically run after an initial clone"
     cd $top/$service
     log=$top/log/$service.setup.log
     echo "Writing to $log";
-    script/setup > $log 2>&1 || echo "Setup failed for $service"
+    script/setup 2>&1 | tee $log || echo "Setup failed for $service"
```

Comment:
I would say leave out the `tee` -- there is way too much noise in the terminal.  You could put a message about how the user can run `tail -f $log` if they want to see it all.